geode-redis/src/main/java/org/apache/geode/redis/internal/executor/RedisKeyInRegion.java

```diff
@@ -46,4 +47,13 @@ public boolean del(ByteArrayWrapper key) {
   public boolean exists(ByteArrayWrapper key) {
     return localRegion.containsKey(key);
   }
+
+  public boolean rename(ByteArrayWrapper oldKey, ByteArrayWrapper newKey) {
+
+    Object value = localRegion.get(oldKey);
```

Comment:
It looks like this impl will not work on String since they do not yet store their data in the region.
It might be best just to wait for the String work to complete